src/main/java/nstarlike/javautils/credential/PasswordUtils.java

```diff
@@ -0,0 +1,120 @@
+package nstarlike.javautils.credential;
+
+import java.nio.charset.StandardCharsets;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class PasswordUtils {
+	public static String getRandomPassword(int size, boolean hasUpper, boolean hasNum, boolean hasSpecial) {
+		//variable declarations
+		StringBuilder password = new StringBuilder();
+		char[] lowers = {'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 
+							'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z'};
+		char[] uppers = {'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M', 'N', 'O', 
+							'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z'};
+		char[] specials = {'~', '!', '@', '#', '$', '%', '^', '&', '*', '(', ')', '-', '+', '?'};
+		List<String> chTypes = new ArrayList<String>();
+		chTypes.add("lower");
```

Comment:
It might be good if you use ENUM

Reply:
Thanks for your review and I will fix it later.